includes/apple-exporter/components/class-body.php

```diff
@@ -137,36 +208,57 @@ protected function html_enabled() {
 	 * @access private
 	 */
 	private function set_default_layout() {
-		$this->json[ 'layout' ] = 'body-layout';
-		$this->register_layout( 'body-layout', array(
-			'columnStart' => $this->get_setting( 'body_offset' ),
-			'columnSpan'  => $this->get_setting( 'body_column_span' ),
-			'margin'      => array(
-				'top' => 12,
-				'bottom' => 12
+		$this->register_layout(
+			'body-layout',
+			'body-layout',
+			array(
+				'columnStart' => $this->get_setting( 'body_offset' ),
+				'columnSpan'  => $this->get_setting( 'body_column_span' ),
 			),
-		) );
+			'layout'
+		);
 
 		// Also pre-register the layout that will be used later for the last body component
-		$this->register_layout( 'body-layout-last', array(
-			'columnStart' => $this->get_setting( 'body_offset' ),
-			'columnSpan'  => $this->get_setting( 'body_column_span' ),
-			'margin'      => array(
-				'top' => 12,
-				'bottom' => 30
-			),
-		) );
+		$this->register_layout(
+			'body-layout-last',
+			'body-layout-last',
+			array(
+				'columnStart' => $this->get_setting( 'body_offset' ),
+				'columnSpan'  => $this->get_setting( 'body_column_span' ),
+			)
+		);
 	}
 
 	/**
-	 * Get the default style for the component.
+	 * Get the default style spec for the component.
 	 *
 	 * @return array
 	 * @access private
 	 */
-	private function get_default_style() {
+	private function get_default_style_spec() {
 		return array(
 			'textAlignment' => 'left',
+			'fontName' => '%%body_font%%',
+			'fontSize' => '%%body_size%%',
+			'tracking' => '%%body_tracking%%',
+			'lineHeight' => '%%body_line_height%%',
+			'textColor' => '%%body_color%%',
+			'linkStyle' => array(
+				'textColor' => '%%body_link_color%%',
+			),
+			'paragraphSpacingBefore' => 18,
```

Comment:
Right, but ideally those values would scale depending on font size rather than being hardcoded. I figured this could be an opportunity to update that logic, since we were already under the hood. Not essential.